src/lib/forms/rjsf/widgets/RJSFVocabulary.js

```diff
@@ -82,13 +91,14 @@ export class RJSFVocabulary extends Component {
   render() {
     const { autofocus, label, placeholder, readonly, required } = this.props;
     const { options, isLoading, value, error } = this.state;
+
     return (
       <Form.Select
         fluid
         selection
         options={options}
         label={label}
-        value={value}
+        value={this.validateValue(value, options) ? value : undefined}
```

Comment:
did you check the documentation of RJSF? they have a prop responsible for validation function

Reply:
ah ok, that escaped my mind

Reply:
yes, true. However in this case we should display an error not simply remove the value